eks-hybrid-iam-ra-setup.sh

```diff
@@ -43,6 +50,17 @@ if ! [ "${DRY_RUN}" = "true" ]; then
     fi
 fi
 
+get_aws_signing_helper_update_command() {
+  local credential_process_from_config
```

Comment:
elsewhere in the file you use four spaces for indent, not two like in this function

Reply:
Fixed indentation, 4 it is